draft-ietf-opsawg-pcapng.md

```diff
@@ -2016,6 +2016,39 @@ The following is a list of Secrets Types.
 {: vspace='0'}
 
 
+{: indent='8'}
+0x55414b4c:
+: OPC UA Key Log.
```

Comment:
You might want to add a link to, for example, https://opcfoundation.org/about/opc-technologies/opc-ua/, to indicate what "OPC UA" is, to add some context.